app/forms/idv/how_to_verify_form.rb

```diff
@@ -0,0 +1,9 @@
+module Idv
+  class HowToVerifyForm
+    include ActiveModel::Model
+
+    def submit(_params)
+      FormResponse.new(success: valid?, errors: errors)
```

Comment:
Does this include an error message for if the use didn't pick either option? eg the example banner in in figma [here](https://www.figma.com/file/sACtyhlHe1PNkrqccMP9eJ/LG-11119-%26-LG-10995-Opt-in-IPP?type=design&node-id=389-15462&mode=design&t=jYE7Zo9ZyKaDcIb8-0)